src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -37,7 +37,7 @@ public InstancesController(IInstanceRepository instanceRepository)
         /// <param name="applicationOwnerId">application owner</param>
         /// <param name="applicationId">application id</param>
         /// <returns>list of all instances for given instanceowner</returns>
-        /// GET api/v1/instances
+        /// GET api/storage/v1/instances
```

Comment:
url chould be: {component}/api/v1/{resource}/{guid}/

Reply:
ya, i have to update the url formats in both altinn core and here after David's code is merged in.

Reply:
GET api/storage/v1/instances should be GET storage/api/v1/instances